client/character.lua

```diff
@@ -334,9 +334,19 @@ local function chooseCharacter()
                         description = Lang:t('info.delete_character_description', { playerName = name }),
                         icon = 'trash',
                         onSelect = function()
-                            TriggerServerEvent('qbx_core:server:deleteCharacter', character.citizenid)
-                            destroyPreviewCam()
-                            chooseCharacter()
+                            local confirmDelete = lib.alertDialog({
+                                header = Lang:t('info.delete_character'),
+                                content = Lang:t('info.confirm_delete'),
+                                centered = true,
+                                cancel = true
+                            })
+                            if confirmDelete == 'confirm' then
```

Comment:
Consider moving this if statement in-line rather than storing the alertDialog in a variable